FAQ.md

```diff
@@ -39,12 +39,7 @@ Intel processors. AMD and ARM processors will be supported in the near future.
 
 ### Can Firecracker be used with Kubernetes, Docker, or Kata containers today?
 
-Not yet. We are making Firecracker open source because it provides a
-meaningfully different approach to security for running containers. We hope that
-others in the communities that build open source container technology find it
-useful. We are working to make Firecracker integrate naturally with the
-container ecosystem, with the goal to provide seamless integration in the future
-to provide more choices in how container workloads are isolated.
+Not yet with Kubernetes and Docker, but in release [1.5.0](https://github.com/kata-containers/runtime/releases/tag/1.5.0) of Kata Containers, support for the Firecracker hypervisor has been included. We hope that others in the communities that build open source container technology find it useful. We are working to make Firecracker integrate naturally with the container ecosystem, with the goal to provide seamless integration in the future to provide more choices in how container workloads are isolated.
```

Comment:
Well, technically it can be used with docker CLI and k8s when configured with Kata. 

There are limitations with Kubernetes use cases though.

Reply:
I think for a FAQ entry it has the right amount of detail. Users of Kata know it integrates with docker and k8.